src/pages/scholarships.js

```diff
@@ -1,9 +1,73 @@
 import Layout from "../components/layout";
+import { createGlobalStyle } from "styled-components";
 import React, { useEffect } from "react";
 import Header from "../components/header";
 import Footer from "../components/footer";
 import afterParty from "../img/after-party.png";
 
+const GlobalStyles = createGlobalStyle`
```

Comment:
not a huge deal but if this and attend share 90% of the same code it might be nice to abstract it into a function that they can both share. Up to you.

Reply:
Componentized!